src/zjs_modules.c

```diff
@@ -255,6 +266,53 @@ static ZJS_DECL_FUNC(process_exit)
     exit(status);
 }
 #endif
+#ifdef ZJS_DYNAMIC_LOAD
+void zjs_modules_check_load_file(char *file)
```

Comment:
It appears this file argument is never used?

It also looks like you might have forgotten to declare this in a header file which might be why that wasn't caught? I would think it would be giving a warning though. I guess that gets through Travis because we're not testing this flag.

Reply:
Indeed, left over code.